meson_options.txt

```diff
@@ -13,9 +12,13 @@ option('plugin_dell', type : 'boolean', value : true, description : 'enable Dell
 option('plugin_dummy', type : 'boolean', value : false, description : 'enable the dummy device')
 option('plugin_synaptics', type: 'boolean', value: true, description : 'enable Synaptics MST hub support')
 option('plugin_thunderbolt', type : 'boolean', value : true, description : 'enable Thunderbolt support')
-option('plugin_uefi_labels', type : 'boolean', value : true, description : 'enable UEFI labels support')
 option('plugin_uefi', type : 'boolean', value : true, description : 'enable UEFI support')
 option('systemd', type : 'boolean', value : true, description : 'enable systemd support')
 option('systemdunitdir', type: 'string', value: '', description: 'Directory for systemd units')
 option('tests', type : 'boolean', value : true, description : 'enable tests')
 option('udevdir', type: 'string', value: '', description: 'Directory for udev rules')
+option('efi-cc', type : 'string', value : 'gcc', description : 'the compiler to use for EFI modules')
+option('efi-ld', type : 'string', value : 'ld', description : 'the linker to use for EFI modules')
+option('efi-libdir', type : 'string', description : 'path to the EFI lib directory')
+option('efi-ldsdir', type : 'string', description : 'path to the EFI lds directory')
```

Comment:
Do you think we should set some defaults here?

Reply:
Fixed

Reply:
Not fixed; setting defaults makes the build fail as the autdetection doesn't get run in `plugins/uefi/efi`...

Reply:
Ah, right forgot about that autodetection stuff.  OK then no concerns.